benchmarks/integer_overflow_and_underflow/overflow_add.yaml

```diff
@@ -0,0 +1,8 @@
+integer_underflow_and_overflow/overflow_add:
+  has_bug: true  # Trail of bits suggests adding a test where has_bug: false
+  issues:
+    - location:
+        - bytecode_offset: 139
+          line_number: 15
+          src: 9:27  # or 9:27:0 to match solc ast?
```

Comment:
I will remove this based on #3

Reply:
I think you are thinking about this as an "and" rather than an "or". See https://github.com/EthereumAnalysisBenchmarks/Omnibus/wiki/Issue-Location for a discussion of what's up.

Ultimately, either way is not critical because whatever is lacking here and needed in the benchmarking tools will be put into the benchmark-reporting-side configuration. The observation though was that this was a place to do it accross multiple kinds of benchmark reporting tools. 

And I'll probably have something similar in correlating outputs from different tools elsewhere as well.

Reply:
https://github.com/EthereumAnalysisBenchmarks/Omnibus/wiki/Issue-Location sums it up pretty well. I think for now lets work with `bytecode_offset` and `line_number`. If there is a demand to identify the expression then we can add that later. Ok?